src/config.ml

```diff
@@ -369,6 +397,26 @@ module Defaults = struct
     |> add Connect_timeout 120
     |> add Connect_retry_max `Unlimited
     |> add Proto (None, `Udp)
+
+  let server_config =
+    let open Conf_map in
+    empty
+    |> add Dev (`Tun ,(Some "tun0"))
+    |> add Ping_interval `Not_configured
+    |> add Cipher "AES-256-CBC"
+    |> add Ping_timeout (`Restart 120)
+    |> add Renegotiate_seconds 3600
+    |> add Bind (Some (Some 1195, None))
+    |> add Handshake_window 60
+    |> add Transition_window 3600
+    |> add Tls_timeout 2
+    |> add Resolv_retry `Infinite
+    |> add Auth_retry `None
+    |> add Connect_timeout 120
+    |> add Connect_retry_max `Unlimited
+    |> add Proto (Some `Ipv4, `Tcp (Some `Server))
+    |> add Tls_mode `Server
+    |> add Server ((Ipaddr.V4.of_string_exn "10.89.0.0"), Ipaddr.V4.Prefix.of_string_exn "10.89.0.0/24")
```

Comment:
- I believe the default device should be `Dev (Tun, None)` ? Does having `Tun` as default here work when making a `tap` server?
- isn't the default port 1194? I think it should be `None, None` here to allow distinguishing between default vs literal port 1194.
- does the server only listen on ipv4 by default? no ipv6?
- Couldn't find any mention of this ip in openvpn. Seems like something we should complain about missing rather than setting by default?